sys/include/time_q48.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2025 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    sys_time_q48        48.16 Unix timestamps
+ * @ingroup     sys
+ *
+ * @{
+ *
+ * @file
+ * @brief       Q48.16 Unix time
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ */
+
+#ifndef TIME_Q48_H
+#define TIME_Q48_H
+
+#include <stdint.h>
+
+#include "time_units.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Epoch Q48.16 time format
+ *
+ * The upper 48 bits of the counter are the seconds since 1st January 1970
+ * The lower 16 bits are the fractions of the second with 1/64k prec
+ */
+typedef uint64_t time_q48_t;
```

Comment:
```suggestion
typedef int64_t time_q48_t;
```

would be able to capture most of our history books (maybe not all the dinosaurs)

but also express the day of birth of every living human.